utility/src/main/java/org/oppia/util/firebase/FirebaseModule.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class FirebaseModule {
+  @Singleton
+  @Provides
+  fun provideFirebaseCrashlytics(): FirebaseCrashlytics {
```

Comment:
Maybe instead provide an implementation of the wrapper/logger class and pass in FirebaseCrashlytics (I'm not sure if we actually want the crashlytics object in the DI graph--we probably want to hide it within one of our implementations to keep future migrations/refactors a bit simpler).

Reply:
Done